sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -94,6 +96,41 @@ FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
 credential.update("{new_key}");
 ```
 
+#### Create a Form Recognizer client using Microsoft identity platform (formerly Azure Active Directory)
+
+Azure SDK for Java supports an Azure Identity package, making it easy to get credentials from Microsoft identity
+platform. First, add the package:
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.3</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+All the implemented ways to request a credential can be found under the `com.azure.identity.credential` package. The
+sample below shows how to use an Azure Active Directory (AAD) application client secret to authorize with Azure Form
+Recognizer.
+
+##### Authorizing with DefaultAzureCredential
```

Comment:
Should this section be similar to the TextAnalytics section we have here - https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/README.md#option-2-create-textanalyticsclient-with-azure-active-directory-credential

Reply:
I am going to copy this change in Text Analytics readme. I revisited eventhub and servicebus readme and found we should telling user about DefaultAzureCrednetial instead of TokenCredential. TokenCredential is interface only.

Reply:
Then why mention TokenCredential here -https://github.com/Azure/azure-sdk-for-java/pull/11156/files/71154c677556c6ec78e88bf398364f1b6da3a34c#diff-49d7f9d8b6def8c99411771de4caf755R126

Reply:
Why having create a TokenCredential object?